src/web/app/components/status-message/status-message.component.html

```diff
@@ -2,5 +2,7 @@
   <button class="close" (click)="dismiss(statusMessage)">
     <span aria-hidden="true">&times;</span>
   </button>
-  <div style="padding-right: 30px;">{{ statusMessage.message }}</div>
+
+  <!--TODO: Change this behavior when no status messages are returned in raw HTML-->
```

Comment:
No need to have this TODO. Supporting status message with raw HTML may be something we actually need to support.